### DISCUSSION TITLE
April Fools

### DISCUSSION THREAD

**Discussion by Shluffy** [theforumhelpers/theforumhelpers.github.io#168] [Q&A] [answered]
So I forked this to set github pages to the af branch, and the header and footer aren't up to date. Are you making a new branch or using a new one for next year, cause the old one could use updates. ([https://shluffy.github.io/theforumhelpers.github.io/](https://shluffy.github.io/theforumhelpers.github.io/))

**Comment by Accio1** [comment, marked as answer]
This branch is not intended to be updated. It serves as an archive of what the site looked like on April Fool's Day this year. While similar ideas may be used on future April Fool's Days, it will be built from a different branch. Therefore, no changes need to be made to the old archive branch.

**Reply by Shluffy** [reply]
Okay. The only problem I was having was that links would got to the main domain, rather then file.

**Reply by Shluffy** [reply]
I edited header so it should work
